### PR TITLE
migrate recipes to RA2

### PR DESCRIPTION
**Pull request by boubou19**

### CODE REVIEW

File: src/main/java/com/github/bartimaeusnek/bartworks/API/BioObjectAdder.java [hunk 1]
```diff
@@ -176,22 +178,14 @@ public static void regenerateBioFluids() {
...
-                        GT_Values.NI,
-                        GT_Values.NI,
-                        null,
-                        500,
-                        120);
+                }
+
+                GT_Values.RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(10)).noItemOutputs()
+                        .fluidInputs(new FluidStack(B.getFluid(), 1000)).fluidOutputs(dnaFluid).duration(15 * SECONDS)
+                        .eut(2).addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/API/BioObjectAdder.java]: duration and eut seem to be wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/FluidLoader.java [hunk 2]
```diff
@@ -93,20 +95,11 @@ public static void run() {
...
-                        GT_Values.NI,
-                        GT_Values.NI,
-                        GT_Values.NI,
-                        GT_Values.NI,
-                        null,
-                        500,
-                        120);
+
+                GT_Values.RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(10)).noItemOutputs()
+                        .fluidInputs(new FluidStack(B.getFluid(), 1000)).fluidOutputs(dnaFluid).duration(15 * SECONDS)
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/FluidLoader.java]: again, time and eut are wrong.

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java [hunk 3]
```diff
@@ -1,170 +1,126 @@
...
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Hydrogen, 1),
-                GT_Values.NI,
-                null,
-                790,
-                120);
+
+        GT_Values.RA.stdBuilder().itemInputs(WerkstoffLoader.Forsterit.get(OrePrefixes.dust, 7))
+                .itemOutputs(
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 2L),
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 1L))
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java:25]: chem balance is wrong here and in all the following. needs to be GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 3L) here, and 3x sio2 in the other recipes as well

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java [hunk 4]
```diff
@@ -1,170 +1,126 @@
...
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 1L),
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Chrome, 3L),
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 3L),
+                        GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Hydrogen, 2))
+                .noFluidInputs().fluidOutputs(Materials.Oxygen.getGas(2000L)).duration(23 * SECONDS)
+                .eut(TierEU.RECIPE_MV).addTo(sElectrolyzerRecipes);
+
+        GT_Values.RA.stdBuilder()
+                .itemInputs(WerkstoffLoader.VanadioOxyDravit.get(OrePrefixes.dust, 53), ItemList.Cell_Empty.get(3))
+                .itemOutputs(
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java:79]: for this one chembalance is messed up far worse. but it starts with the very weird compound formula and should be adjusted there first

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java [hunk 5]
```diff
@@ -1,170 +1,126 @@
...
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Boron, 3),
+                        GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Hydrogen, 3))
+                .noFluidInputs().fluidOutputs(Materials.Oxygen.getGas(19000L)).duration(35 * SECONDS + 10 * TICKS)
+                .eut(TierEU.RECIPE_MV).addTo(sElectrolyzerRecipes);
+
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        WerkstoffLoader.ChromoAluminoPovondrait.get(OrePrefixes.dust, 53),
+                        ItemList.Cell_Empty.get(3))
+                .itemOutputs(
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java:93]: same here. no chembalance, very strange chem formula.

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java [hunk 6]
```diff
@@ -1,170 +1,126 @@
...
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnalium, 6L),
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 6),
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Boron, 3),
+                        GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Hydrogen, 3))
+                .noFluidInputs().fluidOutputs(Materials.Oxygen.getGas(19000L)).duration(36 * SECONDS)
+                .eut(TierEU.RECIPE_MV).addTo(sElectrolyzerRecipes);
+
+        GT_Values.RA.stdBuilder()
+                .itemInputs(WerkstoffLoader.FluorBuergerit.get(OrePrefixes.dust, 50), ItemList.Cell_Empty.get(3))
+                .itemOutputs(
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java:105]: same here. no chembalance, very strange chem formula.

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java [hunk 7]
```diff
@@ -1,170 +1,126 @@
...
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 6L),
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 6),
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Boron, 3),
+                        GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Fluorine, 3))
+                .noFluidInputs().fluidOutputs(Materials.Oxygen.getGas(6000L)).duration(36 * SECONDS + 10 * TICKS)
+                .eut(TierEU.RECIPE_MV).addTo(sElectrolyzerRecipes);
+
+        GT_Values.RA.stdBuilder()
+                .itemInputs(WerkstoffLoader.Olenit.get(OrePrefixes.dust, 51), ItemList.Cell_Empty.get(1))
+                .itemOutputs(
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Electrolyzer.java:117]: same here. no chembalance, very strange chem formula.

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AddSomeRecipes.java [hunk 8]
```diff
@@ -28,64 +30,52 @@ public class AddSomeRecipes implements Runnable {
...
+        GT_Values.RA.stdBuilder().itemInputs(BW_NonMeta_MaterialItems.Depleted_Tiberium_1.get(1))
+                .itemOutputs(
+                        WerkstoffLoader.Zirconium.get(dust),
+                        WerkstoffLoader.Zirconium.get(dust),
+                        WerkstoffLoader.Tiberium.get(dustSmall, 2),
+                        WerkstoffLoader.Zirconium.get(dust, 2),
+                        GT_OreDictUnificator.get(dust, Materials.TungstenSteel, 8L),
+                        GT_OreDictUnificator.get(dust, Materials.Platinum, 1L))
+                .outputChances(100_00, 50_00, 50_00, 25_00, 100_00, 100_00).noFluidInputs()
+                .fluidOutputs(WerkstoffLoader.Xenon.getFluidOrGas(1)).duration(15 * SECONDS).eut(2)
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AddSomeRecipes.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AddSomeRecipes.java [hunk 9]
```diff
@@ -28,64 +30,52 @@ public class AddSomeRecipes implements Runnable {
...
+        GT_Values.RA.stdBuilder().itemInputs(BW_NonMeta_MaterialItems.Depleted_Tiberium_2.get(1))
+                .itemOutputs(
+                        WerkstoffLoader.Zirconium.get(dust, 2),
+                        WerkstoffLoader.Zirconium.get(dust, 2),
+                        WerkstoffLoader.Tiberium.get(dust),
+                        WerkstoffLoader.Zirconium.get(dust, 4),
+                        GT_OreDictUnificator.get(dust, Materials.TungstenSteel, 18L),
+                        GT_OreDictUnificator.get(dust, Materials.Platinum, 2L))
+                .outputChances(100_00, 50_00, 50_00, 25_00, 100_00, 100_00).noFluidInputs()
+                .fluidOutputs(WerkstoffLoader.Xenon.getFluidOrGas(2)).duration(15 * SECONDS).eut(2)
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AddSomeRecipes.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AddSomeRecipes.java [hunk 10]
```diff
@@ -28,64 +30,52 @@ public class AddSomeRecipes implements Runnable {
...
+        GT_Values.RA.stdBuilder().itemInputs(BW_NonMeta_MaterialItems.Depleted_Tiberium_4.get(1))
+                .itemOutputs(
+                        WerkstoffLoader.Zirconium.get(dust, 4),
+                        WerkstoffLoader.Zirconium.get(dust, 4),
+                        WerkstoffLoader.Tiberium.get(dust, 2),
+                        WerkstoffLoader.Zirconium.get(dust, 8),
+                        GT_OreDictUnificator.get(dust, Materials.TungstenSteel, 38L),
+                        GT_OreDictUnificator.get(dust, Materials.Platinum, 4L))
+                .outputChances(100_00, 50_00, 50_00, 25_00, 100_00, 100_00).noFluidInputs()
+                .fluidOutputs(WerkstoffLoader.Xenon.getFluidOrGas(4)).duration(15 * SECONDS).eut(2)
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AddSomeRecipes.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AddSomeRecipes.java [hunk 11]
```diff
@@ -28,64 +30,52 @@ public class AddSomeRecipes implements Runnable {
...
+        GT_Values.RA.stdBuilder().itemInputs(BW_NonMeta_MaterialItems.Depleted_TheCoreCell.get(1))
+                .itemOutputs(
+                        ItemList.Depleted_Naquadah_4.get(8),
+                        WerkstoffLoader.Zirconium.get(dust, 64),
+                        WerkstoffLoader.Zirconium.get(dust, 64),
+                        GT_OreDictUnificator.get(dust, Materials.TungstenSteel, 64L),
+                        GT_OreDictUnificator.get(dust, Materials.TungstenSteel, 64L),
+                        GT_OreDictUnificator.get(dust, Materials.TungstenSteel, 48L))
+                .outputChances(100_00, 50_00, 50_00, 100_00, 100_00, 100_00).noFluidInputs().noFluidOutputs()
+                .duration(15 * SECONDS).eut(2).addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AddSomeRecipes.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java [hunk 12]
```diff
@@ -14,164 +17,99 @@
...
-                BW_Util.getMachineVoltageFromTier(4));
+        GT_Values.RA.stdBuilder().itemInputs(Materials.Thorium.getDust(1))
+                .itemOutputs(
+                        Materials.Thorium.getDust(1),
+                        Materials.Thorium.getDust(1),
+                        WerkstoffLoader.Thorium232.get(OrePrefixes.dust, 1),
+                        WerkstoffLoader.Thorium232.get(OrePrefixes.dust, 1),
+                        WerkstoffLoader.Thorium232.get(OrePrefixes.dust, 1))
+                .outputChances(800, 375, 22, 22, 5).noFluidInputs().noFluidOutputs().duration(15 * SECONDS).eut(2)
+                .addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java [hunk 13]
```diff
@@ -14,164 +17,99 @@
...
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        new ItemStack(GT_TileEntity_THTR.THTRMaterials.aTHTR_Materials, 1, 5),
+                        GT_Utility.getIntegratedCircuit(17))
+                .itemOutputs(new ItemStack(GT_TileEntity_THTR.THTRMaterials.aTHTR_Materials, 64, 6)).noFluidInputs()
+                .noFluidOutputs().duration(40 * MINUTES).eut(TierEU.RECIPE_LV).addTo(sCentrifugeRecipes);
+
+        GT_Values.RA.stdBuilder().itemInputs(new ItemStack(GT_TileEntity_THTR.THTRMaterials.aTHTR_Materials, 1, 6))
+                .itemOutputs(Materials.Lead.getDust(1)).outputChances(300).noFluidInputs().noFluidOutputs()
+                .duration(15 * SECONDS).eut(2).addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java [hunk 14]
```diff
@@ -14,164 +17,99 @@
...
+            GT_Values.RA.stdBuilder()
+                    .itemInputs(new ItemStack(GT_TileEntity_HTGR.HTGRMaterials.aHTGR_Materials, 1, i + 6))
+                    .itemOutputs(
+                            fuel.recycledItems[0],
+                            fuel.recycledItems[1],
+                            fuel.recycledItems[2],
+                            fuel.recycledItems[3],
+                            fuel.recycledItems[4])
+                    .outputChances(fuel.recycleChances).noFluidInputs().fluidOutputs(fuel.recycledFluid)
+                    .duration(15 * SECONDS).eut(2).addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java [hunk 15]
```diff
@@ -14,164 +17,99 @@
...
-                null,
-                null,
-                null,
-                null,
-                60 * 20,
-                BW_Util.getMachineVoltageFromTier(3));
+        GT_Values.RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(17)).itemOutputs(BioItemList.getOther(4))
+                .fluidInputs(new FluidStack(BioCultureLoader.eColi.getFluid(), 1000))
+                .fluidOutputs(new FluidStack(FluidLoader.BioLabFluidMaterials[1], 10)).duration(15 * SECONDS).eut(2)
+                .addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java [hunk 16]
```diff
@@ -14,164 +17,99 @@
...
+
+        GT_Values.RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(17)).noItemOutputs()
+                .fluidInputs(new FluidStack(FluidLoader.BioLabFluidMaterials[1], 1000))
+                .fluidOutputs(new FluidStack(FluidLoader.BioLabFluidMaterials[3], 250)).duration(15 * SECONDS).eut(2)
+                .addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java [hunk 17]
```diff
@@ -14,164 +17,99 @@
...
+
+        GT_Values.RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(17)).noItemOutputs()
+                .fluidInputs(new FluidStack(BioCultureLoader.CommonYeast.getFluid(), 1000))
+                .fluidOutputs(new FluidStack(FluidLoader.BioLabFluidMaterials[2], 10)).duration(15 * SECONDS).eut(2)
+                .addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/recipes/Centrifuge.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java [hunk 18]
```diff
@@ -131,220 +141,163 @@ private PlatinumSludgeOverHaul() {}
...
+                .eut(TierEU.RECIPE_LV).addTo(UniversalChemical);
+
+        GT_Values.RA.stdBuilder().itemInputs(PTConcentrate.get(cell, 2))
+                .itemOutputs(
+                        PTSaltCrude.get(dustTiny, 16),
+                        PTRawPowder.get(dustTiny, 4),
+                        Materials.NitrogenDioxide.getCells(1),
+                        Materials.DilutedSulfuricAcid.getCells(1))
+                .fluidInputs(AmmoniumChloride.getFluidOrGas(400)).fluidOutputs(PDAmmonia.getFluidOrGas(400))
+                .duration(15 * SECONDS).eut(2).addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AdditionalRecipes.java [hunk 19]
```diff
@@ -380,37 +389,30 @@ public static void run() {
...
+        GT_Values.RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(1))
+                .itemOutputs(
+                        Materials.Sugar.getDustSmall(21),
+                        Materials.Calcium.getDustTiny(1),
+                        Materials.Magnesium.getDustTiny(1),
+                        Materials.Potassium.getDustTiny(1),
+                        Materials.Sodium.getDustTiny(4),
+                        Materials.Phosphor.getDustTiny(1))
+                .outputChances(100_00, 100_00, 10_00, 100_00, 10_00, 10_00).fluidInputs(Materials.Milk.getFluid(10000))
+                .fluidOutputs(Materials.Water.getFluid(8832)).duration(15 * SECONDS).eut(2).addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/processingLoaders/AdditionalRecipes.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/werkstoff_loaders/recipe/CrushedLoader.java [hunk 20]
```diff
@@ -102,116 +114,98 @@ public void run(Werkstoff werkstoff) {
...
-                null,
-                null,
-                null,
-                null,
-                new int[] { 10000, 1111 },
-                (int) Math.max(1L, werkstoff.getStats().getMass() * 8L),
-                5);
+        GT_Values.RA.stdBuilder().itemInputs(werkstoff.get(dustImpure))
+                .itemOutputs(werkstoff.get(dust), werkstoff.getOreByProduct(0, dust)).outputChances(100_00, 11_11)
+                .noFluidInputs().noFluidOutputs().duration(15 * SECONDS).eut(2).addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/werkstoff_loaders/recipe/CrushedLoader.java]: time and eut wrong

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/werkstoff_loaders/recipe/CrushedLoader.java [hunk 21]
```diff
@@ -102,116 +114,98 @@ public void run(Werkstoff werkstoff) {
...
+
+        GT_Values.RA.stdBuilder().itemInputs(werkstoff.get(dustPure))
+                .itemOutputs(werkstoff.get(dust), werkstoff.getOreByProduct(1, dust)).outputChances(100_00, 11_11)
+                .noFluidInputs().noFluidOutputs().duration(15 * SECONDS).eut(2).addTo(sCentrifugeRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/werkstoff_loaders/recipe/CrushedLoader.java]: time and eut wrong

**Review by chochem — CHANGES_REQUESTED** [review]
only looked at the code, did not try ingame.
Mostly some big issue with centrifuge recipes.
If you dont want to fix the chem balance stuff, thats fine. at least maybe add a todo comment there.

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/werkstoff_loaders/recipe/BlockLoader.java [hunk 22]
```diff
@@ -17,38 +17,40 @@
...
-            GT_Values.RA.addCutterRecipe(
-                    werkstoff.get(block),
-                    werkstoff.get(plate, 9),
-                    null,
-                    (int) Math.max(werkstoff.getStats().getMass() * 10L, 1L),
-                    30);
+
+            GT_Values.RA.stdBuilder().itemInputs(werkstoff.get(block)).itemOutputs(werkstoff.get(plate, 9))
+                    .noFluidInputs().noFluidOutputs().duration((int) Math.max(werkstoff.getStats().getMass() * 10L, 1L))
+                    .eut(TierEU.RECIPE_LV).addTo(sCutterRecipes);
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/werkstoff_loaders/recipe/BlockLoader.java:52]: should be variants with 3 different fluids (and times) and nothing without fluids. probably applies to all cutter recipes in this PR

Comment (chochem) [reply src/main/java/com/github/bartimaeusnek/bartworks/system/material/werkstoff_loaders/recipe/BlockLoader.java:52]: should hopefully be resolved by https://github.com/GTNewHorizons/GT5-Unofficial/pull/2271

Comment (chochem) [reply src/main/java/com/github/bartimaeusnek/bartworks/system/material/werkstoff_loaders/recipe/BlockLoader.java:52]: though a GT dep update would be needed for it to work in dev env

**Comment by boubou19** [comment]
@chochem i'm not doing chem rebalance in this PR, main goal is to have 1:1 recipes with RA1, so i think i have adressed all your concerns

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java [hunk 23]
```diff
@@ -416,12 +418,13 @@ private static void runHelperrecipes() {
...
                 .eut(TierEU.RECIPE_LV).addTo(UniversalChemical);
 
-        GT_Values.RA.addFluidHeaterRecipe(
-                GT_Utility.getIntegratedCircuit(1),
-                RutheniumTetroxideSollution.getFluidOrGas(800),
-                HotRutheniumTetroxideSollution.getFluidOrGas(800),
-                300,
-                480);
+        GT_Values.RA.stdBuilder()
+            .itemInputs(GT_Utility.getIntegratedCircuit(11))
```
Comment (chochem) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java]: best to keep the circuit the same, or is there a reason not to?

Comment (boubou19) [reply src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java]: likely a typo

**Review by chochem — APPROVED** [review]

**Pull request merged**